src/comments.js

```diff
@@ -5,11 +5,21 @@ const {
   addDanglingComment,
   addTrailingComment,
   getNextNonSpaceNonCommentCharacterIndex,
-  isNextLineEmpty
+  isNextLineEmpty,
+  skipNewline,
+  isPreviousLineEmpty
 } = require("prettier").util;
-const { concat, join, indent, hardline } = require("prettier").doc.builders;
+const {
+  concat,
+  join,
+  indent,
+  hardline,
+  cursor,
+  lineSuffix,
+  breakParent
+} = require("prettier").doc.builders;
 // TODO: remove after resolve https://github.com/prettier/prettier/pull/5049
-const { hasNewline, hasNewlineInRange } = require("./util");
+const { hasNewline, hasNewlineInRange, isLookupNode } = require("./util");
```

Comment:
Let's take this out of `prettier.util` once #868 is merged

Reply:
:+1: Yep, i think we can implement and fix all problem in this/next week and release new version in next week too, (Fix unnecessary todo in my todo list :smile: )

Reply:
Awesome, can't wait! :tada: